README.md

```diff
@@ -27,7 +25,7 @@ cd ~/whitepaper
 
 Compile using pandoc
 ```bash
-pandoc whitepaper.md --latex-engine=xelatex -o whitepaper.pdf
+pandoc --verbose whitepaper.md --latex-engine=xelatex -o whitepaper.pdf --dpi=300
```

Comment:
@plink01001 

read the reference please. 
with a quick read you would not ask such trivial questions.

Reply:
Hi @EvertonMelo 

I had checked the manual prior to asking you. Unfortunately, it does not provide very good info.

```
verbose
Give verbose debugging output. Currently this only has an effect with PDF output.
```

```
dpi=NUMBER
Specify the dpi (dots per inch) value for conversion from pixels to inch/centimeters and vice versa. The default is 96dpi. Technically, the correct term would be ppi (pixels per inch).
```

The definitions are fairly self-evident, but it gives no insight into why the values you are submitting in the PR are better than the defaults.

We really appreciate your contribution and want to work with you to get it merged. We just want to make sure that the choices to change are well documented.

Can you please provide a brief description of what the benefit is / reason behind changing both of those values is?

Reply:
ok, first benefit for 300 DPI:

![dpi-examples](https://user-images.githubusercontent.com/7637553/39049179-c10ee028-4476-11e8-9817-0a00ec103836.jpg)

300 DPI are best quailify for Document with images, but so 300 DPI for Photo

but  300 DPI for photo are quality small.

Reply:
Secund benefit Mr @plink01001,

--verbose; 

this helps in the visualization in the compilation process.

Reply:
third Benefit Mr @plink01001 

&& and -Y 

less iteration, how to install these packages is something that the user decided to do, because not to make it as autonomous as possible?

the operators provides this.